app/controllers/question.rb

```diff
@@ -0,0 +1,17 @@
+get '/questions' do
+  @questions = Question.all
+  erb :'/questions/index'
+end 
+
+get '/questions/new' do
+  erb:'/questions/new'
+end
+
+
+get '/questions/:id' do 
+  @question = Question.find_by(id: params[:id])
+  @answers = Answer.where(question_id: params[:id])
+  erb :'/questions/show'
+end 
```

Comment:
created route and views page so users can view questions, their answers, and comments for both of them